Packs/Core/Playbooks/playbook-Excessive_User_Account_Lockouts_README.md

```diff
@@ -0,0 +1,67 @@
+This playbook is designed to handle the following alerts:
+
+- Excessive user account lockouts
+- Excessive account lockouts on suspicious users
+- Excessive user account lockouts from a suspicious source
+
+The playbook investigates and responds to excessive user account lockout alerts. It gathers information about the alert, enriches relevant host data, and analyzes event patterns. This analysis helps distinguish between benign lockouts and lockouts caused by brute-force or password spray attacks.
+
+Playbook Stages:
+
+Triage:
+
+- The playbook enriches the alert with details about the lockout events.
+
+Investigation:
```

Comment:
better formatting
Investigation:                                                                                                                                                                                         │
│ - The playbook analyzes the lockout event timestamps to detect patterns.                                                                                                                               │
│ - Checks for related medium severity brute-force alerts in the incident.                                                                                                                               │
│ - Retrieves the Risk Score for the Caller Computer that caused the lockouts.